src/components/ui/contributors.tsx

```diff
@@ -9,7 +11,7 @@ export function Contributors() {
         data-octo-click="hovercard-link-click"
         data-octo-dimensions="link_type:self"
       >
-        <img
+        <Image
```

Comment:
[Kown Browser Bugs for next/image
](https://nextjs.org/docs/api-reference/next/image#known-browser-bugs)

[Safari 15+](https://bugs.webkit.org/show_bug.cgi?id=243601) displays a gray border while loading. Possible solutions:
- Use CSS @supports (font: -apple-system-body) and (-webkit-appearance: none) { img[loading="lazy"] { clip-path: inset(0.6px) } }
- Use [priority](https://nextjs.org/docs/api-reference/next/image#priority) if the image is above the fold

[Firefox 67+](https://bugzilla.mozilla.org/show_bug.cgi?id=1556156) displays a white background while loading. Possible solutions:
- Enable [AVIF formats](https://nextjs.org/docs/api-reference/next/image#acceptable-formats)
- Use [placeholder="blur"](https://nextjs.org/docs/api-reference/next/image#placeholder)